app/v1/connectors/RetrieveEmploymentsExpensesConnector.scala

```diff
@@ -35,11 +34,17 @@ class RetrieveEmploymentsExpensesConnector @Inject() (val http: HttpClient, val
       ec: ExecutionContext,
       correlationId: String): Future[DownstreamOutcome[RetrieveEmploymentsExpensesResponse]] = {
 
-    val url = s"income-tax/expenses/employments/${request.nino.nino}/${request.taxYear}?view=${request.source.toDownstream}"
+    val source  = request.source.toDownstream
```

Comment:
I think is visually better to `import request._`

Reply:
I had it like that originally, but I wanted to shorten the lengths of the URL lines by pulling out `nino.value` and `source.toDownstream` into variables (as they are the same in both TYS and non TYS), and by that point the `import request._` didn't really serve much purpose.